packages/@adobe/spectrum-css-temp/components/table/index.css

```diff
@@ -150,6 +150,10 @@ governing permissions and limitations under the License.
 }
 
 .spectrum-Table-cellContents {
+  display: flex;
```

Comment:
not sure we want this, it makes an assumption about what people will put in here
were you running into any issues not having it?
I'm assuming it had to do with justification of contents?

Reply:
did you have a chance to try doing it with text align?

Reply:
Yeah, I added the display + justify-content so that the cell contents would still respect the Column's alignment setting. I could use `text-align: inherit` but I would have to change the wrapper to have the proper `text-align` applied to in via `react-spectrum-Table-cell--alignCenter`. Would that be better than this `display: flex`?

Reply:
so my only hesitation here is that flex children have different defaults than non-flex children
we might have some extra documentation to do if this is what we want to do, afterall, it's easy enough to break out of the flex by creating a wrapper div around your content that doesn't display flex

since this component isn't in stable release yet, I think I'm ok with saying this is how we want to do it

Reply:
hrm, bit of a snag here. `text-overflow` doesn't work on flex containers, but using `text-align` will only work if the contents of the Cell are text and not a View like in the story included